build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
 ext {
-  jettyVersion = "11.0.8"
+  jettyVersion = "11.0.10"
```

Comment:
Would recommend using `11.0.11` as it contains an important fix for suffix based url-patterns.